exercises/robot-simulator/.meta/version

```diff
@@ -1 +1 @@
-2.2.0
+2.3.0
```

Comment:
Since I opened the issue this appears to have been bumped again to 3.0.0. So it would be great if you could change this to 3.0.0 🙂

Reply:
No problem